src/lib/presets.ts

```diff
@@ -82,5 +82,11 @@ export const presets: Record<string, PresetConfig> = {
 		owner: 'huntabyte',
 		repo: 'shadcn-svelte',
 		glob: ['**/sites/docs/src/content/**/*.md']
+	},
+	'Remult': {
```

Comment:
This goes in the URL, so let's make this lowercase
```suggestion
	'remult': {
```